docs/plugins/ingest.asciidoc

```diff
@@ -474,8 +474,10 @@ PUT _ingest/pipeline/my-pipeline-id
 --------------------------------------------------
 // AUTOSENSE
 
-NOTE: Each ingest node updates its processors asynchronously in the background, so it may take a few seconds for all
-      nodes to have the latest version of the pipeline.
+NOTE: The put pipeline api also instructs all ingest nodes to reload their in-memory representation of pipelines, so that
+      pipeline changes take in effect immediately. Also each ingest node asynchronously checks for pipeline changes in the
+      background. How often processors are updated in the background can be controlled via the
+      `ingest.pipeline.store.update.interval` setting and defaults is `30s`.
```

Comment:
this needs to be updated if we go for removing the background updater completely.